src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -493,8 +493,9 @@ static int do_map()
     }
 
     if (forker.is_parent()) {
+      global_init_postfork_start(g_ceph_context);
```

Comment:
Crashing is one thing (which should have been tested before the breaking commit was merged), but passing all the tests is another (rbd-nbd workunit and fax in nbd mode).